.github/workflows/rust.yml

```diff
@@ -17,6 +17,22 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Build
-      run: cd near-rust-allocator-proxy && cargo build --verbose
+      run: RUSTFLAGS='-D warnings' cargo build --verbose --workspace
```

Comment:
for GHA, it's better to just set the env at the top-level: 

```
env:
  CARGO_TERM_COLOR: always
  RUSTFLAGS: '-D warnings'
```

I also like throwing `CARGO_INCREMENTAL: 0` and `RUST_BACKTRACE: short` in there

Reply:
@matklad Thanks, I added it to: https://github.com/near/near-memory-tracker/pull/86